src/main/scala/cognite/spark/v1/AssetsRelation.scala

```diff
@@ -6,7 +6,7 @@ import cats.effect.IO
 import cats.implicits._
 import cognite.spark.v1.PushdownUtilities._
 import cognite.spark.v1.SparkSchemaHelper._
-import com.cognite.sdk.scala.common.{WithExternalId, WithId}
+import com.cognite.sdk.scala.common._
```

Comment:
necessary to be able to import LabelsOnUpdate transformers from scala-sdk/package.scala.